### DISCUSSION TITLE
Is it possible to make this theme two-column?

### DISCUSSION THREAD

**Discussion by shingjan** [mmistakes/jekyll-theme-basically-basic#111] [Q&A] [answered]
## Summary

To make this theme theme two-column using the existing navigation bar?

## Motivation

This theme is absolutely stunning! I love it so much and am gonna use it for my personal page. I just wander what I could do to make this page two column using the existing navigation bar. Like make it sticky instead of popping out/in. Also need to make the page a little balanced with the sticky side bar.

**Comment by mmistakes** [comment, marked as answer]
Not unless you're comfortable hacking the theme's stylesheets to do what you want. I purposely designed it not to be two columns.

If you're looking for a two column I'd suggest trying out my [**Minimal Mistakes** theme](https://github.com/mmistakes/minimal-mistakes) instead... or to a lesser extent [**So Simple**](https://github.com/mmistakes/so-simple-theme).
